From f7919a4360ebb5e668dfd5851e7d93d853e96492 Mon Sep 17 00:00:00 2001
From: Tim Snyder <snyder.tim@gmail.com>
Date: Sat, 30 Dec 2023 01:18:44 +0000
Subject: [PATCH] pkgconfig files need to consistently report version also

---
 configure | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/configure b/configure
index 0e08db5f..f55d8faa 100755
--- a/configure
+++ b/configure
@@ -590,8 +590,8 @@ MAKEFLAGS=
 # Identity of this package.
 PACKAGE_NAME='SystemC'
 PACKAGE_TARNAME='systemc'
-PACKAGE_VERSION='3.0.0'
-PACKAGE_STRING='SystemC 3.0.0'
+PACKAGE_VERSION='3.0.0_pub_rev_2023129'
+PACKAGE_STRING='SystemC 3.0.0_pub_rev_2023129'
 PACKAGE_BUGREPORT='http://forums.accellera.org/forum/9-systemc/'
 PACKAGE_URL='http://accellera.org/'
 
-- 
2.43.0

